build/travis/job2_AppImage/build.sh

```diff
@@ -27,17 +29,31 @@ else
   makefile_overrides="" # use Makefile defaults
 fi
 
-# Build MuseScore AppImage inside Docker image
-docker run -i -v "${PWD}:/MuseScore" library/centos:6 \
-   /bin/bash -c "/MuseScore/build/Linux+BSD/portable/Recipe $makefile_overrides"
+# Build AppImage. Are we cross-compiling?
+case "$1" in
+  --32bit )
+    shift
+    # Build MuseScore AppImage inside 32-bit Docker image
+    docker run -i -v "${PWD}:/MuseScore" toopher/centos-i386:centos6 /bin/bash -c \
+      "linux32 --32bit i386 /MuseScore/build/Linux+BSD/portable/Recipe $makefile_overrides"
+    ;;
+  * )
+    [ "$1" == "--64bit" ] && shift || true
+    # Build MuseScore AppImage inside native (64-bit) Docker image
+    docker run -i -v "${PWD}:/MuseScore" library/centos:6 /bin/bash -c \
+      "/MuseScore/build/Linux+BSD/portable/Recipe $makefile_overrides"
+    ;;
+esac
```

Comment:
@ericfont Keep an eye on [this issue](https://github.com/probonopd/AppImageKit/issues/91) for AppImageKit where RazZziel has succeeded in building AppImages for ARM on Travis. He's building on the [feature/32bit_builds_docker](https://github.com/probonopd/AppImageKit/tree/feature/32bit_builds_docker) branch. Build logs are [here](https://travis-ci.org/probonopd/AppImageKit/builds). He only had to change [.travis.yml](https://github.com/probonopd/AppImageKit/blob/d93068948814528007742c17eb599150917e3dfd/.travis.yml#L6-L12) to make ARM builds work for AppImageKit (MuseScore will probably be a bit harder because you'll have to find the dependencies for the OS you choose).